README.md

```diff
@@ -401,6 +406,7 @@ Your contributions and suggestions are heartily♥ welcome. (✿◕‿◕). Plea
 * [ZoomEye](https://www.zoomeye.org/) - Search engine for cyberspace that lets the user find specific network components.
 * [gOSINT](https://github.com/Nhoya/gOSINT) - OSINT tool with multiple modules and a telegram scraper.
 * [Amass](https://github.com/caffix/amass) - Subdomain enumeration via scraping, web archives, brute forcing, permutations, reverse DNS sweeping, TLS certificates, passive DNS data sources, etc.
+* [Hunter.io](https://hunter.io/) - Data broker providing a Web search interface for discovering the email addresses and other organizational details of a company.
```

Comment:
I am not sure if we want to add paid services with very limited options as a free service. I am open to your advice here as you probably have more experience with this.

Reply:
That's fair. Hunter.io provides a "freemium" service with several tiers offering increasing amounts of data for each successive tier, including (roughly):

1. anonymous visitor/logged out/unregistered
1. free registration
1. paid registration

I'm unsure what the threshold for inclusion is or should be, but I have no strong feelings one way or another. I would be just as happy to remove Hunter.io from the list as I would be to include it. If you're hesitant, I would say we should just leave it off for now as there is plenty of additional information in the OSINT category already. :)

Reply:
Fair enough. Thanks for the segmentation of the tier. I guess we can have it here as long as the freemium is useful (which it seems to be).